integration-tests/features/steps/common.py

```diff
@@ -163,11 +163,16 @@ def set_job_status(context, id, status):
     context.response = requests.put(url)
 
 
+@when("I reset status for the job service")
 @when("I set status for job service to {status}")
-def set_job_status(context, status):
+def set_job_status(context, status=None):
     endpoint = "{jobs_api_url}api/v1/service/state".format(
                jobs_api_url=context.jobs_api_url)
-    url = "{endpoint}?state={status}".format(endpoint=endpoint, status=status)
+    if status is not None:
+        url = "{endpoint}?state={status}".format(endpoint=endpoint,
+                                                 status=status)
+    else:
+        url = endpoint
```

Comment:
Just a nitpick - this could be more readable:

```python
url = "{jobs_api_url}api/v1/service/state".format(jobs_api_url=context.jobs_api_url)
if status is not None:
    url = "{url}?state={status}".format(url=url, status=status)
```